tests/idris2/basic049/expected

```diff
@@ -151,7 +151,7 @@ Fld:226:1--226:10
 
 Error: While processing right hand side of r2_shouldNotTypecheck1. Ambiguous elaboration. Possible results:
     Main.R2.MkR
-    Main.R1.MkR Type
+    Main.R1.MkR ?field
```

Comment:
This looks suspicious  Can this PR influence badly on type inference?

Reply:
I think the old output is wrong here. There's no reason for the name to be resolved to a `Type`, and I think it's probably because we need to have `Context` in another place in `Error`.

Reply:
I suppose I should probably test this theory while I have the chance...

Reply:
...and that is indeed what's going on. There's a few places in the `Error` structure that don't record the context at the point of error. It should be there whenever there's a `Term`. If it's not, we have to be sure the term has been normalised.

Anyway, the updated output is correct. I'll fix the error message display separately.